appveyor.yml

```diff
@@ -37,8 +37,8 @@ environment:
       GOVERSION: 115
       SQLINSTANCE: SQL2019
     - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2019
-      GOVERSION: 115
-      SQLINSTANCE: SQL2017
```

Comment:
SQL2017 is already covered at https://github.com/denisenkom/go-mssqldb/blob/045585d74f9069afe2e115b6235eb043c8047043/appveyor.yml#L25
so I dropped this case.

Reply:
From what I can see every matrix entry needs a `SQLINSTANCE` since there's no global setting :thinking: